src/cleaners/asValue.js

```diff
@@ -1,8 +1,15 @@
-export function asValue(value) {
+export function asValue(...values) {
+  let message = 'Expected '
+  if (values.length !== 1) message += 'one of: '
+  for (let i = 0; i < values.length; ++i) {
+    if (i > 0) message += ' | '
+    message += JSON.stringify(values[i])
+  }
```

Comment:
Can we do this lazily? Or perhaps just move it to line 13. I feel like we should only be computing the error message when the cleaner fails.

Reply:
Moved to the later area, as suggested.